system/jlib/jptree.hpp

```diff
@@ -293,7 +296,7 @@ inline static bool isValidXPathChr(char c)
     return ('\0' != c && (isalnum(c) || strchr(validChrs, c)));
 }
 
-jlib_decl IPropertyTree * loadArgsIntoConfiguration(IPropertyTree *config, const char * * argv);
+jlib_decl IPropertyTree * loadArgsIntoConfiguration(IPropertyTree *config, const char * * argv, std::initializer_list<std::string> ignoreOptions = {});
```

Comment:
picky: initializer_list<const char *> would allow constant data to be passed.

Reply:
changed.